codecov.yml

```diff
@@ -0,0 +1,25 @@
+codecov:
+  require_ci_to_pass: no
+
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
```

Comment:
So we need 70% coverage?

Reply:
no and yes, I have it set to not fail for now. Once we get tests going properly I will activate the code coverage rules

Reply:
Create a devops ticket in the backlog for activation of code coverage rules so we can still push this then :)

Reply:
done #37